examples/awaits.rb

```diff
@@ -1,9 +1,13 @@
 # frozen_string_literal: true
 
+# For use with bundler:
+# require 'rubygems'
+# require 'bundler/setup'
+
 require 'discordrb'
 
 # Create a bot
-bot = Discordrb::Bot.new token: 'token'
+bot = Discordrb::Bot.new token: 'your_bot_token'
```

Comment:
This is the format the other examples use, I'm not sure why this one was different to begin with.
```suggestion
bot = Discordrb::Bot.new token: 'B0T.T0KEN.here'
```

Reply:
Sounds good.